content/media-player/tracking/setup.md

```diff
@@ -0,0 +1,767 @@
+
+> **❓ Version 1 vs Version 2 media schemas**
+>
+> There are two versions of schemas for media events that our trackers may use to track media events. We recommend adopting the v2 schemas, but you may choose v1 schemas in case you want use a tracking SDK that doesn’t yet support v2 schemas and you don’t need the additional features of v2 schemas.
+>
+> They differ in the following ways:
+>
+> 1. V2 schemas enable more accurate metrics about the playback (e.g., played duration, average playback rate) that are computed on the tracker instead of estimated in modeling.
+> 2. V2 schemas enable tracking ads played during media playback.
+> 3. V2 schemas provide additional events to track playback quality and periodic media pings during playback.
+> 4. V2 schemas support live streaming video (in addition to VOD).
+
+| Tracking SDK | Platform | Media schemas version | Provides auto-tracking? | Player |
+| --- | --- | --- | --- | --- |
+| https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/javascript-trackers/browser-tracker/browser-tracker-v3-reference/plugins/media/ | Web | v2 | No | Any |
+| https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/javascript-trackers/browser-tracker/browser-tracker-v3-reference/plugins/vimeo-tracking/ | Web  | v2 | Yes | Vimeo |
+| https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/mobile-trackers/tracking-events/media-tracking/ | Mobile (iOS) | v2 | No | Any |
+| https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/mobile-trackers/tracking-events/media-tracking/ | Mobile (iOS) | v2 | Yes | AVPlayer |
+| https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/mobile-trackers/tracking-events/media-tracking/ | Mobile (Android) | v2 | No | Any |
+| https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/javascript-trackers/browser-tracker/browser-tracker-v3-reference/plugins/media-tracking/ | Web | v1 | Yes | HTML5 media or video |
+| https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/javascript-trackers/browser-tracker/browser-tracker-v3-reference/plugins/youtube-tracking/ | Web | v1 | Yes | YouTube |
+
+#### **Step 2:** Install the tracking SDK in your app
+
+Once you have chosen the tracking SDK to use, the next step is to install it in your app.
+
+{{< tabs groupId="tracking sdk" >}}
+
+{{% tab name="Media JS plugin" %}}
+
+Install the `@snowplow/browser-tracker` and `@snowplow/browser-plugin-media` via npm, yarn or any other package manager of your choice. Example using `npm`:
+
+```bash
+npm install @snowplow/browser-tracker @snowplow/browser-plugin-media
+```
```

Comment:
I am thinking that in order to simplify the instructions, we should drop the instructions for using the JS plugins using a script tag. I think that is a less used path and it is quite similar to using via NPM.

So I would just add a note in here under the JS plugins that it is possible to install with a script tag and remove the script tag sections. Is that okay?

```
{{% notice note %}}
It is also possible to integrate the plugin using a JavaScript script tag. For instructions, [please visit the documentation](https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/javascript-tracker-v3/plugins/media/).
 {{% /notice %}}
```

Reply:
Yeah that simplifies things!